xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -846,6 +848,14 @@ void CDVDVideoCodecAndroidMediaCodec::SetDropState(bool bDrop)
     m_videobuffer.iFlags &= ~DVP_FLAG_DROPPED;
 }
 
+void CDVDVideoCodecAndroidMediaCodec::SetCodecControl(int flags)
+{
+  if (m_codecControlFlags != flags)
+    if (g_advancedSettings.CanLogComponent(LOGVIDEO))
```

Comment:
@koying 
maybe you fooled yourself with this? Is the first if missing backets by any chance?